modules/olm-mirroring-catalog.adoc

```diff
@@ -81,14 +81,14 @@ $ oc adm catalog mirror \
     <mirror_registry>:<port> \//<2>
     [-a ${REG_CREDS}] \//<3>
     [--insecure] \//<4>
-    [--filter-by-os="<os>/<arch>"] \//<5>
```

Comment:
Normally I would sort the flags such that the optional ones are all shown last, but in this case I didn't want to reorder/number everything differently if this will eventually get fixed.

Reply:
Not sure if the lack of brackets `[]` in this flag is intentional but calling it out just in case.

Reply:
Intentional because `[]` indicates an optional flag.